config/global.php

```diff
@@ -166,8 +166,15 @@
                 $ipsResolved[] = $ip;
             } else {
                 $ipFromHost = @gethostbyname($ip);
-                if (!empty($ipFromHost)) {
+                if (!empty($ipFromHost)
```

Comment:
the extra filter var check is needed as ipFromHost will be the hostname if it couldn't resolve it.

![image](https://user-images.githubusercontent.com/273120/134281703-0e94751b-d454-4558-9556-ead1eb3736ea.png)